osu.Game/Online/API/Requests/Responses/Comment.cs

```diff
@@ -40,11 +38,16 @@ public class Comment
         [JsonProperty(@"votes_count")]
         public int VotesCount { get; set; }
 
+        public CommentableType CommentableType;
+
         [JsonProperty(@"commenatble_type")]
```

Comment:
Must have missed this before - how is/was this working?
```suggestion
         [JsonProperty(@"commentable_type")]
```

Reply:
I never used it so yeah, good catch